consumer/auth.go

```diff
@@ -193,7 +193,7 @@ func (s *State) CreateUser(c *gin.Context) {
 		panic("wtf")
 	}
 	if err := s.Db.AutoMigrate(&gateway.UserModel{}); err != nil {
-		log.Print("the error is: %v", err)
```

Comment:
We want to use log, not fmt. The logger include other valuable info such as date and time, unlike fmt

Reply:
okaay .. i just got an error when i performed the testing that says (Print call has possible formatting directive %v) and when i searched about it i figured that Print does not support formatting directives.

Reply:
Nope that is for %# you added in a latter place